_includes/figure.liquid

```diff
@@ -1,4 +1,6 @@
-{% assign img_path = include.path | remove: '.jpg' | remove: '.jpeg' | remove: '.png' | remove: '.tiff' | remove: '.gif' %}
+{% assign parts = include.path | split: '.' %}
```

Comment:
This will lead to unexpected results if the image name have other `.` than the extension (e.g. `aaa.bbb.jpg`).

Reply:
done